app/models/merchant.rb

```diff
@@ -1,4 +1,5 @@
 class Merchant <ApplicationRecord
+  has_many :users
```

Comment:
ok so i migrated :0 Im pretty sure in DTR we talked about getting that approved but I spaced and rolled with it. My bad. Im trash but tests are passing.